pyprof/tests/L0/run_pyprof_nvtx/test_pyprof_nvtx.py

```diff
@@ -0,0 +1,573 @@
+#!/usr/bin/env python3
```

Comment:
if this change is unrelated to this PR, can it be removed and saved for a separate PR?

Reply:
This change was due to the rebase.  It should be gone now.